addons/website/static/src/active_elements/plausible_push.js

```diff
@@ -0,0 +1,18 @@
+import { Component, useAttachedEl } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+
+class PlausiblePush extends Component {
+    static selector = ".js_plausible_push";
+    
+    setup() {
+        const el = useAttachedEl();
+        const {eventName, eventParams} = el.dataset;
```

Comment:
Original code did default `eventParams` to `{}`. Could it be that JQ's `.data()` did parse the dataset entry into JSON ?